asset/js/setup.js

```diff
@@ -32,6 +32,34 @@ function setupWithSearch(siteData) {
   VueStrap.installEvents(vm);
 }
 
+/**
+ *   site-nav-btn function to toggle itself and site navigation elements.
+ */
+// eslint-disable-next-line no-unused-vars
+function siteNavToggle(siteNavBtn) {
+  siteNavBtn.classList.toggle('morph');
+  document.getElementById('site-nav').classList.toggle('open');
+  document.getElementById('site-nav-btn-wrap').classList.toggle('open');
+}
+
 jQuery.getJSON(`${baseUrl}/siteData.json`)
   .then(siteData => setupWithSearch(siteData))
   .catch(() => setup());
+
+/**
+ * Creates event listener for all dropdown-btns in page.
+ */
+Array.prototype.forEach.call(document.getElementsByClassName('dropdown-btn'),
```

Comment:
- Single-line comment since this isn't a function definition.

  ```js
  // Creates event listener for all dropdown-btns in page.
  ```

- Why are we not doing the following?

  ```js
  document.getElementsByClassName('dropdown-btn').forEach((element) => {
  ```

Reply:
> Why are we not doing the following?

This [post](https://stackoverflow.com/questions/3871547/js-iterating-over-result-of-getelementsbyclassname-using-array-foreach) explains it. I have found this [updated solution](https://stackoverflow.com/questions/22754315/for-loop-for-htmlcollection-elements) to be a better alternative, using: 

```javascript
const dropdownBtnList = document.getElementByClassName('dropdown-btn');
for (let dropdownBtn of dropdownBtnList ) { ... }
```